app/src/main/java/com/duhapp/dnotes/features/manage_category/ui/ManageCategoryViewModel.kt

```diff
@@ -1,12 +1,14 @@
 package com.duhapp.dnotes.features.manage_category.ui
 
```

Comment:
Not silindiği zaman Repository de lastDetedItem olarak entity yi saklayalim. Undo use case inde repository den isUndoAvaiable fonksiyonunu çağıralım fonksiyon true dönerse repository den undo fonksiyonunu çağıralım değilse hata dönelim.

Reply:
Silmeyelim de silme tarihi mi tutalım